doc/kci_data.md

```diff
@@ -125,34 +125,32 @@ having an API token for the KernelCI staging API.  This was useful to be able
 to show the results on the web dashboard.  The same command can be run with a
 local instance using `--db-config=localhost`.
 
-## Creating user
+## Creating new API user (experimental)
 
-We can create new users of KernelCI-API using `kci_data`.
+We can create users for the new [KernelCI API](/docs/api) using `kci_data`.
+This is still under development and it's not used in production by KernelCI.
 
-We need to provide an API token to access `/user` endpoint of API.
-Please refer the instructions [here](https://kernelci.org/docs/api/api-details/#create-an-api-token-with-security-scopes) to create token for admin user.
+An API token with admin permissions is required in order to do this, please see
+the documentation about [how to create
+one](/docs/api/api-details/#create-an-api-token-with-security-scopes).
 
-For security purpose, instead of providing `--db-token` from the comman line we can add `db_token` in settings file i.e. `kernelci.conf`.
+For security reasons, instead of providing `--db-token` from the command line
+we can add `db_token` in the [settings file](../settings) e.g. `kernelci.conf`:
 
-```
+```ini
 [db:api]
```

Comment:
```suggestion
[db:docker-host]
```

Reply:
This is coming from the `kernelci-pipeline` YAML configuration, which is not something `kernelci-core` depends on. The dependency goes the other way round. The code in `kernelci-core` for supporting the new API is still incomplete, and so the documentation will also have some gaps for a while.

As discussed in a meeting today, providing sample commands that work with the pipeline YAML config should be part of the "getting started" pipeline documentation page. The standard `kci_data` documentation page should have generic examples that are agnostic of the code in `kernelci-pipeline`. So I don't think these latest suggestions should be applied. Some tweaks could (should) be made in the pipeline documentation to provide similar commands that actually work though, so I'll take a look at that next.

Reply:
But we should provide all the information needed to use this particular functionality of kci_data.

Reply:
I think the page explains how to use the `kci_data create_user`, or is it missing something? The pipeline "getting started" guide should explain how to do this in the particular case of the pipeline which provides a YAML file with `docker-host`. I've sent a PR to cover that, it's now on staging: https://static.staging.kernelci.org/docs/api/getting-started/#create-an-admin-user-account

Reply:
Okay, so I think we need to provide a link here to pipeline documentation containing configurations related to this functionality.

Reply:
Again, this is going in the wrong direction. The kernelci-core tools don't depend on kernelci-pipeline. There is a link from the "getting started" page to the `kci_data` page as a reference, in addition to sample commands. The code in `kernelci-core` is not complete yet and so is the documentation, but that's fine.